packages/vhd-lib/src/Vhd/VhdDirectory.js

```diff
@@ -0,0 +1,206 @@
+import { BLOCK_UNUSED } from '../../dist/_constants'
+import { createLogger } from '@xen-orchestra/log'
+import { fuFooter, fuHeader, checksumStruct } from '../_structs'
+import { SECTOR_SIZE } from '../_constants'
+import { buildHeader, buildFooter, computeBatSize } from './_utils'
+import { test, set as setBitmap } from '../_bitmap'
+import { VhdAbstract } from './VhdAbstract'
+import assert from 'assert'
+
+const { debug } = createLogger('vhd-lib:VhdDirectory')
+
+// ===================================================================
+// Directory format
+// <path>
+// ├─ header // raw content of the header
+// ├─ footer // raw content of the footer
+// ├─ bat // bit array. A zero bit indicates at a position that this block is not present
+// ├─ parentLocator{0-7} // data of a parent locator
```

Comment:
Let's stay as close to the spec as possible :slightly_smiling_face: 

```suggestion
// ├─ parentLocatorEntry{1-8} // data of a parent locator
```

Reply:
As we discussed, let's keep numbering starting from `0` as it's customary in JS.

Reply:
Either rename to `parentLocatorEntry` or resolve this comment if you want to keep using `parentLocator`, I don't mind.